REQUIRE

```diff
@@ -1,2 +1,3 @@
-julia 0.6
-Compat 0.59
\ No newline at end of file
+julia 0.7
+MacroTools
```

Comment:
Can we do this without MacroTools? It's only used in a couple of places for checking input forms, which can be done by hand.

Reply:
In order to solve https://github.com/invenia/Mocking.jl/issues/61
using MacroTools is probably going to be the best way.
I think it would be better to go the otehr way and remove more of the hand checking code and rewrite it to be more readable using MacroTools.